ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -59,6 +59,9 @@ router bgp {{ router_bgp.as }}
 {%             if router_bgp.peer_groups[peer_group].timers is defined and router_bgp.peer_groups[peer_group].timers is not none %}
    neighbor {{ peer_group }} timers {{ router_bgp.peer_groups[peer_group].timers }}
 {%             endif %}
+{%             if router_bgp.peer_groups[peer_group].route_map_in is defined and router_bgp.peer_groups[peer_group].route_map_in is not none %}
```

Comment:
please refactor to use the new `is arista.avd.defined` test

Reply:
Fixed in commit a47a8fb.